entity-services/src/main/xdmp/entity-services/entity-services-codegen.xqy

```diff
@@ -543,12 +543,12 @@ declare function es-codegen:version-translator-generate(
     <convert-instance>
 (:~
  : Creates a map:map instance representation of the target
- : entity type {$entity-type-name} from an envelope document that
- : contains the source entity instance.
- : @param $source  Either an entity-services envelope document, or
- :  an XML node holding the canonical form of the source entity type.
- : @return A map:map instance that holds the data for the target
- :  entity type.
+ : entity type {$entity-type-name} from XML that
+ : contains instance data of {$entity-type-name}, version {$source-version}.
```

Comment:
I liked "an envelope document" over "XML" and would like a mix of old and new like this. Let me know what you think:

```
: entity type {$entity-type-name} from an envelope document that
: contains the source entity instance ie instance data of {$entity-type-name}, version {$source-version}.
```
Since it is only clear from codegen.xqy file that the version is a source version, but wont be the case in the output. So having those words "source entity instance" seems apt.

Other edits are OK.